packages/host/app/components/search-sheet/index.gts

```diff
@@ -175,22 +246,39 @@ export default class SearchSheet extends Component<Signature> {
       </div>
       <SearchInput
         @bottomTreatment={{this.inputBottomTreatment}}
-        @value={{this.searchInputValue}}
+        @value={{this.searchKey}}
         @placeholder='Enter search term or type a command'
         @onFocus={{@onFocus}}
-        @onInput={{fn (mut this.searchInputValue)}}
+        @onInput={{this.setSearchKey}}
       />
       <div class='search-sheet-content'>
+        {{! @glint-ignore Argument of type 'string' is not assignable to parameter of type 'boolean' }}
```

Comment:
Maybe it’s not worth fixing but maybe it could be by changing [this](https://github.com/cardstack/boxel/blob/8e47aadb328cc47b7a00a636f9bb21ea2a970e63/packages/host/app/helpers/truth-helpers.ts#L10) to expect a falsy value vs boolean